wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -468,8 +466,12 @@ private void check() {
                                 MasternodePeerDiscovery discovery = new MasternodePeerDiscovery(mnlist);
                                 peers.addAll(Arrays.asList(discovery.getPeers(services, timeoutValue, timeoutUnit)));
                                 if(peers.size() < 10) {
-                                    log.info("DMN peer discovery returned less than 10 nodes.  Adding seed peers to the list to increase connections");
-                                    peers.addAll(Arrays.asList(seedPeerDiscovery.getPeers(services, timeoutValue, timeoutUnit)));
+                                    if (Constants.NETWORK_PARAMETERS.getAddrSeeds() != null) {
+                                        log.info("DMN peer discovery returned less than 10 nodes.  Adding seed peers to the list to increase connections");
+                                        peers.addAll(Arrays.asList(seedPeerDiscovery.getPeers(services, timeoutValue, timeoutUnit)));
+                                    } else {
+                                        log.info("DMN peer discovery returned less than 10 nodes.  Unable to add seed peers (it is not specified for this network).");
+                                    }
```

Comment:
The above change was added in order to avoid NPE for network parameters with no specified AddrSeed (i.e. DevNet and RegNet).

Reply:
Good Catch.